pipelines/controller/src/main/java/com/google/fhir/analytics/DataProperties.java

```diff
@@ -109,6 +109,12 @@ public class DataProperties {
 
   public String sinkUserName;
 
+  private boolean mapToGoldenResources;
+
+  private String mdmResourceList;
+
+  private String structureDefinitionsDir;
```

Comment:
**question:** What does this have to do with your MDM work?

Reply:
Nada